sdk/core/Azure.Core/src/Internal/SpanOfCharsExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    internal static class SpanOfCharsExtensions
+    {
+        public static void AppendString(this ref Span<char> span, ref int offset, string str)
+        {
+#if NET6_0_OR_GREATER
```

Comment:
Just out of curiosity why net simply `#if NET`?

Reply:
Harder to read (you need to remember that `NET` is 5.0+), also some features are available only in 6.0+ and we don't release for 5.0